chirho/dynamical/internals/solver.py

```diff
@@ -22,6 +22,11 @@ class Solver(Generic[T], pyro.poutine.messenger.Messenger):
     @typing.final
     @staticmethod
     def _pyro_simulate(msg: dict) -> None:
+        msg["done"] = True
+
+    @typing.final
+    @staticmethod
+    def _pyro_post_simulate(msg: dict) -> None:
```

Comment:
This move from `_pyro_simulate` to `_pyro_post_simulate` seems unnecessary, per my comment on `LogTrajectory`.